index.js

```diff
@@ -11,6 +11,9 @@ const connect = require('./lib/mongo_client').connect;
 connect.MongoError = core.MongoError;
 connect.MongoNetworkError = core.MongoNetworkError;
 connect.MongoTimeoutError = core.MongoTimeoutError;
+connect.MongoParseError = core.MongoParseError;
+connect.MongoWriteConcernError = core.MongoWriteConcernError;
+connect.BulkWriteError = require('./lib/bulk/common').BulkWriteError;
```

Comment:
since we plan on eventually changing this to `MongoBulkWriteError`, can we do an alias here?

```js
connect.BulkWriteError = require('./lib/bulk/common').BulkWriteError;
connect.MongoBulkWriteError = connect.BulkWriteError;
```

Reply:
sure thang

Reply:
fixed